inc/components/class-image.php

```diff
@@ -364,7 +369,7 @@ public function get_srcset() : string {
 
 			// Add retina source to srcset, if applicable.
 			if ( is_numeric( $descriptor ) ) {
-				if ( $this->config['retina'] ) {
+				if ( $this->config['retina'] && ( empty( $params['retina'] ) || $params['retina'] ) ) {
```

Comment:
`if ( $this->config['retina'] && empty( $params['retina'] ?? '' ) )`

Try this out, a little more verbose (I really love `??`)